website/scripts/ros/ros_host_vehicle_cmd.js

```diff
@@ -65,3 +65,60 @@ function subscribeToVehicleCMD()
         }
     });
 }
+
+/**
+ * /hardware_interface/speed_pedals
+ */
+function subscribeToSpeedPedals()
+{
+    var listener = new ROSLIB.Topic({
+        ros: g_ros,
+        name: "/hardware_interface/speed_pedals",
+        messageType: "automotive_platform_msgs/SpeedPedals"
+    });
+    listener.subscribe(function(message)
+    {
+        //Check ROSBridge connection before subscribe a topic
+        IsROSBridgeConnected();
+        if(message!=null && message.brake !=null)
+        {
+              //Brake(0-1 percent)
+              if(message.brake != null && g_brakeCircle != null)
+              {
+                let max, brakeLimit = '';
+                //set brake  limit to the host vehicle deceleration(brake) limit if exist in session, otherwise default to 6
+                if(message.brake>0)
+                    brakeLimit = session_hostVehicle.brakeLimit;
+                  
+                max =  brakeLimit != null && brakeLimit.length > 0? brakeLimit: 6;
+                let value = message.brake * max; //brake value
+                updateBrake(g_brakeCircle,max,value);
+              }
+        }
+    });
+}
+/**
+ * /hardware_interface/steering_wheel 
+ */
+function subscribeToSteeringWheel()
+{
+    var listener = new ROSLIB.Topic({
+        ros: g_ros,
+        name: "/hardware_interface/steering_wheel",
+        messageType: "automotive_platform_msgs/SteerWheel"
+    });
+    listener.subscribe(function(message)
+    {
+        //Check ROSBridge connection before subscribe a topic
+        IsROSBridgeConnected();
+        //Steering wheel
+        if(message.angle != null)
+        {
+            let rad = message.angle * 180/(Math.PI);
+            let rotateDegree = rad % 360; //-35 to +35
```

Comment:
What does this comment //-35 to +35 represent? Based on this calculation it wouldn't be the angular range. 

I assume in this calc you are trying to map the rotation to a range of 0 - 360 degrees. While this is fine for your wheel image display, this should not be the numeric value your report. A steering wheel can actually be rotated more than 360 degrees and these values have meaning since it means the wheels are rotated more. I would recommend leaving the displayed value unchanged from what is reported after converting to degrees. 

If you wanted to display a percentage than you will also need the maximum steering angle of the wheel which will be different between cars and requires us to give the UI access to that configuration.

Reply:
@msmcconnell 
The -35/35(- counter clock wise, +clock wise) was the maximum degree when I subscribe to the vehicle commands topics.

Reply:
Addressed https://github.com/usdot-fhwa-stol/carma-web-ui/commit/787924e088b208be4906ac4c821b7ddab3e316bf#diff-75ae112862ffed6c037c6854f5a4bf20